src/components/Table.tsx

```diff
@@ -82,12 +84,13 @@ export default function Table({
           type: 'inactive',
         });
       }
+      console.info(attendance);
```

Comment:
did you mean to leave this here?

Reply:
Nope, just sloppy. Thanks for the catch.